src/utils/convert.js

```diff
@@ -876,6 +899,13 @@ const renderIndex = legacyApp => {
 
     templateContext.REQUIRES = importLines.join('\n');
 
+    const searchOrCreates = findSearchOrCreates(legacyApp);
+    if (_.isEmpty(searchOrCreates)) {
+      templateContext.SEARCH_OR_CREATES = null;
+    } else {
+      templateContext.SEARCH_OR_CREATES = JSON.stringify(searchOrCreates, null, 2);
```

Comment:
super minor, but maybe worth moving the `2` a `spacing` variable somewhere so we can change it all in one go if we ever need to.

Reply:
I don't follow you on this one. Could you elaborate?

Reply:
I meant something like

```js
const JSON_SPACING = 2;
// ...
JSON.stringify(searchOrCreates, null, JSON_SPACING);
```

but it turns out we're only calling `stringify` twice in the file, so it's not a big deal.